src/pytorch_lightning/cli.py

```diff
@@ -200,7 +200,7 @@ def __init__(
         self,
         parser: LightningArgumentParser,
         config: Namespace,
-        config_filename: str,
+        config_filename: str = "config.yaml",
```

Comment:
The `config.yaml` conflicts in some cases. Should we change the default, e.g. could be something like `cli_config.yaml`.

Reply:
I don't have a strong preference here, I'd even say keeping it `config.yaml` is preferable since a change here would also be a breaking change

Reply:
Agree with Ota.